server/src/main/java/org/elasticsearch/cluster/metadata/Metadata.java

```diff
@@ -1448,7 +1448,7 @@ public Metadata build() {
                         visibleClosedIndices.add(name);
                     }
                 }
-                indexMetadata.getAliases().keysIt().forEachRemaining(allAliases::add);
+                indexMetadata.getAliases().keysStream().forEach(allAliases::add);
```

Comment:
I think this has no real benefits and runs at a slightly critical time - so would prefer to just keep this one as is to avoid the wrapper here.